### PR TITLE
Update Zowe SDKs to v8 & FTP CLI plugin to v3

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Update Zowe SDKs to v8 & FTP CLI plugin to v3

## Release Notes

Milestone: v3 GA

Changelog:
- Update Zowe SDKs to `8.0.0` for technical currency.
- Update z/OS FTP Plug-in for Zowe CLI dependency to `3.0.0`.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking change which adds or improves functionality)
- [ ] Breaking change (a change that would cause existing functionality to not work as expected)
- [ ] Documentation (Markdown, README updates)
- [x] Other (please specify above in "Proposed changes" section)

## Checklist

**General**

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] All PR dependencies have been merged and published (if applicable)
- [ ] A GIF or screenshot is included in the PR for visual changes
- [ ] The pre-publish command has been executed:
  - **v2 and below:** `yarn workspace vscode-extension-for-zowe vscode:prepublish`
  - **v3:** `pnpm --filter vscode-extension-for-zowe vscode:prepublish`

**Code coverage**

- [ ] There is coverage for the code that I have added
- [ ] I have added new test cases and they are passing
- [ ] I have manually tested the changes

**Deployment**

- [ ] I have added developer documentation (if applicable)
- [ ] Documentation should be added to Zowe Docs
  - If you're an outside contributor, please post in the [#zowe-doc Slack channel](https://openmainframeproject.slack.com/archives/CC961JYMQ) to coordinate documentation.
  - Otherwise, please check with the rest of the squad about any needed documentation before merging.
- [ ] These changes may need ported to the appropriate branches (list here):

### CODE REVIEW

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fix issue with extender profiles not being included in fresh team configuration file. [#3122](https://github.com/zowe/zowe-explorer-vscode/issues/3122)
 - Fixed issue where file extensions were removed from data sets, causing language detection to sometimes fail for Zowe Explorer extenders. [#3121](https://github.com/zowe/zowe-explorer-vscode/issues/3121)
 - Fixed an issue where copying and pasting a file/folder in the USS tree would fail abruptly, displaying an error. [#3128](https://github.com/zowe/zowe-explorer-vscode/issues/3128)
-- To fix Strange behaviour of Job label under Job Favorites. [#2632](https://github.com/zowe/zowe-explorer-vscode/issues/2632)
```
Comment (JillieBeanSim) [inline packages/zowe-explorer/CHANGELOG.md:17]: removed, a v2 port changelog

File: samples/uss-profile-sample/package.json [hunk 2]
```diff
@@ -46,7 +46,7 @@
     "typescript": "^5.1.3"
   },
   "peerDependencies": {
-    "@zowe/zos-files-for-zowe-sdk": ">=8.0.0-next.0 <8.0.0",
-    "@zowe/zos-uss-for-zowe-sdk": ">=8.0.0-next.0 <8.0.0"
+    "@zowe/zos-files-for-zowe-sdk": ">=8.0.0 <9.0.0",
+    "@zowe/zos-uss-for-zowe-sdk": ">=8.0.0 <9.0.0"
```
Comment (zFernand0) [inline samples/uss-profile-sample/package.json:50]: It should be safe to use `^8.0.0` here 😋

**Review by zFernand0 — DISMISSED** [review]

**Review by traeok — DISMISSED** [review]
LGTM, thanks Billie for updating the SDKs!

**Review by zFernand0 — DISMISSED** [review]

**Review by zFernand0 — DISMISSED** [review]

**Review by traeok — DISMISSED** [review]

File: samples/uss-profile-sample/package.json (hunk 2 above)
Comment (JillieBeanSim) [reply samples/uss-profile-sample/package.json:50]: would `^8.0.0` only cover v8 patches and require an update with 8.1.0 where `>=8.0.0` will cover any 8.x release?

Comment (JillieBeanSim) [reply samples/uss-profile-sample/package.json:50]: I was wondering about adding ^8.0.0 to actual dependencies though to get the patch fixes automatically, what do you all think?

Comment (JillieBeanSim) [reply samples/uss-profile-sample/package.json]: updates in commit https://github.com/zowe/zowe-explorer-vscode/pull/3146/commits/a3c3c1cd977a703224ca58636e1f81fa424a2664

**Review by zFernand0 — APPROVED** [review]

**Review by traeok — APPROVED** [review]

**Review by t1m0thyj — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]

**Review by traeok — APPROVED** [review]

**Pull request merged**